linera-service/tests/end_to_end_tests.rs

```diff
@@ -45,8 +45,21 @@ impl FungibleApp {
             account_owner.to_value()
         );
         let response_body = self.0.query(&query).await.unwrap();
-        serde_json::from_value(response_body["accounts"]["entry"]["value"].clone())
-            .unwrap_or_default()
+        let deserialized_amount =
+            serde_json::from_value::<Amount>(response_body["accounts"]["entry"]["value"].clone());
+        match deserialized_amount {
+            Ok(amount) => amount,
+            Err(e)
+                if e.to_string()
+                    .contains("invalid type: null, expected a string")
```

Comment:
This one is a bit too fragile for me. How about `let amount_option = serde_json::from_value::<Option<Amount>>(..).unwrap();` ?

Reply:
Ah yeah, much better